crates/nu-command/src/filesystem/cd.rs

```diff
@@ -54,7 +54,7 @@ impl Command for Cd {
                         (cwd.to_string_lossy().to_string(), v.span)
                     }
                 } else {
-                    let path = match nu_path::canonicalize_with(&v.item, &cwd) {
+                    let path = match nu_path::canonicalize_with(&v.item.replace('\n', ""), &cwd) {
```

Comment:
for Windows would this also fix `\r\n`? I wonder if we should just trim the whitespace off the end when we `cd` (or maybe also when we `open`?)

cc @fdncred for thoughts

Reply:
hmm, good catch. I think it doesn't fix the \r\n but we could add.
Is there any other case we need to replace?
"open" sounds like a good candidate for the fix.

Reply:
actually, once it will be used across commands should we split that into a different function? so all the other commands can just use the function

Reply:
As long as it is cross platform it sounds good to me. Should we use [trim_end_matches](https://doc.rust-lang.org/src/core/str/mod.rs.html#2175) with chars 0x09..=0x0d aka `is_whitespace()` chars?

Reply:
sounds good!
@fdncred I added the match for those.